File: Python/Beginners/soma_imp_consecutivosII.py
# Caio Cesar 01/12/23
# BEE 1099

casos = int(input())

for i in range(casos):
    numeros = input().split()
    num_x = int(numeros[0])
    num_y = int(numeros[1])

    soma = 0
    if num_x > num_y:
        for n in range(num_y + 1, num_x):
            if n % 2 != 0:
                soma += n
    elif num_y > num_x:
        for n in range(num_x + 1, num_y):
            if n % 2 != 0:
                soma += n
    
    print(soma)

File: C++/Beginner/soma_imp_consecutivosII.cpp
// Caio Cesar 01/12/23
// BEE 1113

#include <iostream>
using namespace std;

int main(){
    int casos, num_x, num_y, soma;
    int i, n;

    cin >> casos;

    for(i = 0; i < casos; i++){
        cin >> num_x >> num_y;

        soma = 0;
        if(num_x > num_y){
            for(n = num_y + 1; n < num_x; n++){
                if(n % 2 != 0){
                    soma += n;
                }
            }
        }else if(num_y > num_x){
            for(n = num_x + 1; n < num_y; n++){
                if(n % 2 != 0){
                    soma += n;
                }
            }
        }

        cout << soma << endl;
    }

    return 0;
}